### PR TITLE
Cargobucket (new)

### PR DESCRIPTION
**Pull request by nyozzl**
- add: New ship Cargobucket expedition salvage vessel
image

image
Why / Balance
I decided a satellite would be cool to make as a salvage vessel and its just another expedition ship which could be used more I think

### CODE REVIEW

**Comment by nyozzl** [comment]
woops forgot the picture

**Comment by dvir001** [comment]
File saved in the wrong folder, please use the folder under _NF

**Comment by nyozzl** [comment]
which file is wrong cause the prototype one is under nf

**Comment by dvir001** [comment]
> which file is wrong cause the prototype one is under nf

You answered your own question.

**Comment by nyozzl** [comment]
right think I fixed it

**Comment by nyozzl** [comment]
yea this is my first map will fix it in a bit

**Comment by nyozzl** [comment]
I don't know what to put in maintainer info

File: Resources/Prototypes/_NF/Shipyard/cargobucket.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+- type: vessel
```
[the remark this answers is not in the document]
Comment (VividPups) [reply Resources/Prototypes/_NF/Shipyard/cargobucket.yml:11]: Just to Clarify,  Maintainer Info is about the person who will Maintain the ship or as in the Maintainer that Approve the ship

**Comment by nyozzl** [comment]
I'm not happy with this I'm probably gonna redesign it in the future

**Pull request closed without merging**